### PR TITLE
KP-7522 Rest of the docs

### PR DESCRIPTION
**Pull request by aajarven**
Add the parts of the documentation that were not completed before Tampere.

Even the rich diff view does not show all the formatting properly (maybe the red/green changed indicator on the side breaks things?) so I recommend reviewing from https://github.com/CSCfi/kielipankki-nlf-harvester/tree/KP-7522/docs and https://github.com/CSCfi/kielipankki-nlf-harvester/blob/KP-7522/docs/versioning.md.

### CODE REVIEW

**Comment by aajarven** [comment]
TODO:
- [ ] Add info about the `unzip` error messages

File: docs/README.md [hunk 1]
```diff
@@ -16,12 +17,26 @@ Approximate figures as of 2023:
...
+
+
 ## Accessing the dataset
 
-### Puhti
+### The newest version on Puhti
 
 For immediate access on the shared file system, the newest dataset is kept in the directory `/scratch/project_2006633/nlf-harvester/zip/`. That directory should be accessible to all users on Puhti.
 
+> [!NOTE]
```
Comment (Traubert) [inline docs/README.md:35]: This can be removed now.

File: docs/README.md [hunk 2]
```diff
@@ -16,12 +17,26 @@ Approximate figures as of 2023:
...
+
+The METS files are also not 1-to-1 match to the public data. Currently we attempt to download all image files whose `fileGrp`'s `USE` is either `Images` or `reference` and `ID` is `IMGGRP` or `ACIMGGRP`. Both available and unavailable ALTO files look the same in the METS, so currently they are downloaded based on the access image page numbers. The scanned page files on the file system can be matched to their METS metadata by matching the number in file name to the `SEQ` attribute in the corresponding `file` element in METS. If you suspect that some data is missing, please contact [the Language Bank of Finland](https://www.kielipankki.fi/support/contact-us/).
+
+At the moment, the harvesting does not automatically produce a list of files that were not available for download, but it will be added later.
```
Comment (Traubert) [inline docs/README.md:26]:
```suggestion
At the moment, the harvesting does not automatically produce a list of files that were not available for download, but one will be made available later.
```

File: docs/README.md [hunk 3]
```diff
@@ -36,15 +51,17 @@ The resulting target directories will have segmented paths, like `1/16/163/1631/
...
+Currently the only way to see if that has happened is to check the last edit time of the zip files by running `ls -l /scratch/project_2006633/nlf-harvester/zip/` on Puhti: if it is before your job started, your results have not been affected by an update.
 
-Versions of the data set will also be made available as [`restic`](https://restic.net/) backups on [Allas](https://docs.csc.fi/data/Allas/introduction/). `restic` commands can be used to list old versions, and to extract either entire snapshots or single `.zip`s.
+### Previous versions in Allas
 
-The retention policy of the previous versions is still open and the Allas versions of the data set have not yet been published.
+Versions of the data set will also be made available as [`restic`](https://restic.net/) backups on [Allas](https://docs.csc.fi/data/Allas/introduction/). The retention policy of the previous versions is still open, so don't rely on old versions being available in the long term.
 
-## Versioning
+See [versionig.md](versioning.md) for more information about previous versions of the dataset and how to access them.
```
Comment (Traubert) [inline docs/README.md:63]:
```suggestion
See [versioning.md](versioning.md) for more information about previous versions of the dataset and how to access them.
```

**Review by Traubert — COMMENTED** [review]
The dividing into files is good, I may have more comments once I've tested the restic stuff.

File: docs/versioning.md
Comment (Traubert) [inline docs/versioning.md:1]: General comment about this file: I think it needs to start with a quickstart, something the user can immediately paste into a terminal and get going. From what I can tell, this should work:

```bash
$ export RESTIC_REPOSITORY="s3:https://a3s.fi/nlf-harvester-versioning"
$ export RESTIC_PASSWORD="nlf-data-at-csc"
$ restic snapshots --no-lock
```

But actually something goes wrong:

```bash
$ restic snapshots --no-lock
Fatal: unable to open config file: Stat: The access key ID you provided does not exist in our records.
Is there a repository at the following location?
s3:https://a3s.fi/nlf-harvester-versioning
```

Is that name right?

**Pull request merged**